wagesscript.sh

```diff
@@ -0,0 +1,13 @@
+#QUESTION 1
```

Comment:
1.	Your file isn’t quite sorted correctly, this would do it: cut -d , -f 1,2 wages.csv | grep -v "gender" | sort -t , -k1,1 -k2,2n | uniq | tr ',' ' '
a.	0.8
2.	1
3.	Not quite the right difference between 12 and 16 years of school. You need to add a call to the sort function in both the pipelines for var1 and var2 in order to sort the wages and then pull the minimum for both those with 12 and 16 years of school. The difference should be ~$4.
a.	.7

2.5/3